tv_apps/tv-ftu/js/tv_ftu.js

```diff
@@ -11,35 +12,92 @@ window.addEventListener('load', function() {
       this.APP_USAGE_ENABLE_VALUE = true;
       this.APP_USAGE_DISABLE_VALUE = false;
 
-      this._noButton =
-          document.getElementById('app-usage-no-button');
-      this._yesButton =
-          document.getElementById('app-usage-yes-button');
-      this._terms =
-          document.getElementById('app-usage-terms');
-      this._termsSection =
-          document.getElementById('app-usage-terms-section');
-      this._arrowIcon =
-          document.getElementById('app-usage-arrow-icon');
+      this._currentPage = null;
+      this._currentContentSection = null;
+      this._currentContent = null;
+      this._currentArrowIcon = null;
+      this._currentFocused = null;
+      this._currentNavigationTarget = null;
 
       this.keyNavAdapter = new KeyNavigationAdapter();
       this.keyNavAdapter.init();
       this.keyNavAdapter.on('move', this.onKeyMove.bind(this));
+      this.keyNavAdapter.on('back-keyup', this.onBackKeyUp.bind(this));
       this.keyNavAdapter.on('enter-keyup', this.onEnterKeyUp.bind(this));
 
-      this.buttonNav = new SimpleKeyNavigation();
-      this.buttonNav.start(
-        [this._noButton, this._yesButton],
-        SimpleKeyNavigation.DIRECTION.HORIZONTAL
-      );
-      this.buttonNav.on('focusChanged', (elem) => {
-        this.fire('focus', elem);
-      });
-      this.buttonNav.pause();
-
-      this._termsSection.focus();
-      this._currentFocused = this._termsSection;
+      this.keyNav = new SimpleKeyNavigation();
+      this.keyNav.start([], SimpleKeyNavigation.DIRECTION.HORIZONTAL);
+      this.keyNav.on('focusChanged', elem => this.fire('focus', elem));
+
+      // Special treatment for translation with links
+      var links = document.querySelectorAll(
+        '#app-usage-page p[data-l10n-id=fxos-is-free] > a');
+      links[0].setAttribute('tabindex', -1);
+      links[0].setAttribute('id', 'fxos-privacy-notice-link');
+      links[1].setAttribute('tabindex', -1);
+      links[1].setAttribute('id', 'moz-privacy-policy-link');
+
       this.on('focus', this.onFocus.bind(this));
+      this.goToPage('app-usage-page');
+    },
+
+    goToPage: function (id) {
+      if (this._currentPage) {
+        this._currentPage.classList.add('hidden');
+      }
+      this._currentPage = document.getElementById(id);
+      this._currentArrowIcon = this._currentPage.querySelector('.arrow-icon');
+      this._currentContent = this._currentPage.querySelector('.page-content');
+      this._currentContentSection =
+        this._currentPage.querySelector('.content-section');
+      this._currentPage.classList.remove('hidden');
+
+      this._currentNavigationTarget = null;
+      this.changeNavigationTarget('buttons');
+      if (this.getContentUnscrolledHeight() > 0) {
+        this.changeNavigationTarget('unscrolled_content');
+      }
+    },
+
+    getContentUnscrolledHeight: function () {
```

Comment:
`Unscrolled Height` is really confusing me. I can't even find this word in dictionary. Until I read the usage below and finally realize it stands for remaining height of the scroll area. Can you find a better naming for it?